src/Command/Dev/UpdateComposer/clear_module_requirements.php.tpl

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
The file needs to have ".tpl" (non-php) extension. Otherwise Symfony framework will include it as part of scanning the namespace https://github.com/magento/ece-tools/blob/develop/config/services.xml#L9